Homework_8/Task_3.py

```diff
@@ -0,0 +1,9 @@
+result_link = 'https://www.youtube.com/watch?v=some_id&list=some_list&index=6&t=0s'
+dict_count = {}
+
+for dict in result_link:
+    if dict in dict_count:
+        dict_count[dict] += 1
+    else:
+        dict_count[dict] = 1
+print(f'How many times an element is repeated in a line: {dict_count}')
```

Comment:
Decision can be simpler
![image](https://github.com/senya2006/Semen-Homework/assets/51053483/797f6326-74e8-42f6-b066-28fe3760d9ff)